### PR TITLE
Update ITEM.tsv

### PR DESCRIPTION
**Pull request by seejy**
Slight changes to spelling, grammar etc

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -4040,7 +4040,7 @@ ITEM_20150323_004039	{memo X}
 ITEM_20150323_004040	{memo X} 
 ITEM_20150323_004041	{memo X} 
 ITEM_20150323_004042	Sleeping Hauberk's Seal Fragment 
-ITEM_20150323_004043	{memo X}A portion of the torn soul of the Demon Master Hauberk. 
+ITEM_20150323_004043	{memo X}A portion of the torned soul of the Demon Master Hauberk.
```
Comment (ttgmichael) [inline ITEM.tsv]: torned isn't a word. torn is the already the past tense form of tear. ^^

**Comment by terrapin47** [comment]
Just as a note, any line that begins with "{memo X}" is an unused line, so there's not much use in editing them :smile:

**Comment by seejy** [comment]
Oh okay. Thanks for the heads up :)

**Comment by ttgmichael** [comment]
Also it'd be great if you could fix up any comments made on your PR and the merge conflict in the meantime ^^.

**Comment by seejy** [comment]
Oh nooo I didn't see that comment. Apologies.

File: ITEM.tsv [hunk 2]
```diff
@@ -5296,46 +5296,46 @@ ITEM_20150714_005295	Petrified the advanced team's relics
...
 ITEM_20150714_005329	Improved Time Bomb 
 ITEM_20150714_005330	{memo X}A powerful bomb that can make a big fuss after a certain time. 
 ITEM_20150714_005331	Sticky Fluids 
 ITEM_20150714_005332	Processed Bomb 
 ITEM_20150714_005333	Hardened Explosives 
 ITEM_20150714_005334	Royal Army Propaganda 
-ITEM_20150714_005335	{memo X}A propaganda to encourage the surrender sprayed by Royal Army when the rebellion, 600 years ago 
-ITEM_20150714_005336	part of the soul of torn monarch Hauberk. 
+ITEM_20150714_005335	{memo X}A propaganda to encourage the surrender sprayed by Royal Army during the rebellion 600 years ago 
+ITEM_20150714_005336	Part of a soul of torned from monarch Hauberk.
```
Comment (ttgmichael) [inline ITEM.tsv]: A part of the Demon Lord Hauberk's torn soul.

**Pull request closed without merging**